modules/nw-operator-cr.adoc

```diff
@@ -345,8 +384,48 @@ If you set this field to `true`, you do not receive the performance benefits of
 |`object`
 |You can control IP forwarding for all traffic on OVN-Kubernetes managed interfaces by using the `ipForwarding` specification in the `Network` resource. Specify `Restricted` to only allow IP forwarding for Kubernetes related traffic. Specify `Global` to allow forwarding of all IP traffic. For new installations, the default is `Restricted`. For updates to {product-title} 4.14, the default is `Global`.
 
+|`ipv4`
+|`object`
+|Optional: Specify an object to configure the internal OVN-Kubernetes masquerade address for host to service traffic for IPv4 addresses.
+
+|`ipv6`
+|`object`
+|Optional: Specify an object to configure the internal OVN-Kubernetes masquerade address for host to service traffic for IPv6 addresses.
+
+|====
+
+[id="gatewayconfig-ipv4-object_{context}"]
+.`gatewayConfig.ipv4` object
+[cols=".^2,.^2,.^6a",options="header"]
+|====
+|Field|Type|Description
+
+|`internalMasqueradeSubnet`
+|`string`
+|
+The masquerade IPv4 addresses that are used internally to enable host to service traffic. The host is configured with these IP addresses as well as the shared gateway bridge interface. The default value is `169.254.169.0/29`.
+
+|====
+
+[id="gatewayconfig-ipv6-object_{context}"]
+.`gatewayConfig.ipv6` object
+[cols=".^2,.^2,.^6a",options="header"]
 |====
+|Field|Type|Description
+
+|`internalMasqueradeSubnet`
+|`string`
+|
+The masquerade IPv6 addresses that are used internally to enable host to service traffic. The host is configured with these IP addresses as well as the shared gateway bridge interface. The default value is `fd69::/125`.
 
+|====
```

Comment:
Lines 421-428 are not in the original PR.

Reply:
sorted this over slack. good to go